runtime/include/jcltracepoints.h

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2025
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+ /* These definitions required by j9.h are in the OpenJ9 jni.h, but needed when OpenJDK jni.h is used. */
```

Comment:
Please remove the leading space.

This file needs to be idempotent:
```c
#if !defined(JCLTRACEPOINTS_H)
#define JCLTRACEPOINTS_H
[... etc. ...]
#endif /* !defined(JCLTRACEPOINTS_H) */
```

Reply:
Hmm, I had rewritten that comment but Eclipse lost my change somehow. Updated.